routers/index.js

```diff
@@ -682,7 +682,8 @@ router.post('/secret/shell',urlencodedParser,authenticateToken,function(req, res
    return  res.send("sorry No user defined" + ser)
  }
 else {
-  return   res.send("test")
+  const uns = serialize.unserialize(req.body)    // Insecure Deserailiazation => Remote Code Execution
```

Comment:
#### Automated PR Comment From coverity
<!-- 0005a7e4c537991b9edf471e344c3a67 -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - Unsafe deserialization
_High CWE-502_
The application performs deserialization by invoking the dangerous `node-serialize.unserialize()` function. If the application passes untrusted data to this function, an attacker can use specialized payloads to achieve remote code execution (RCE).
### How to fix
Modify the application to use a deserialization function that enforces strict type constraints. The actual function used to replace `node-serialize.unserialize()` will depend on the data being serialized and deserialized.